Makefile

```diff
@@ -128,9 +131,35 @@ setup-bare: clean e2e.namespace
 e2e:
 	go test -v $(MOD_FLAGS)  -failfast -timeout 150m ./test/e2e/... -namespace=openshift-operators -kubeconfig=${KUBECONFIG} -olmNamespace=openshift-operator-lifecycle-manager -dummyImage=bitnami/nginx:latest -ginkgo.flakeAttempts=3
 
-e2e-local: build-linux build-wait build-util-linux
-	. ./scripts/build_local.sh
-	. ./scripts/run_e2e_local.sh $(TEST)
+### Start: End To End Tests ###
+
+# Phony prerequisite for targets that rely on the go build cache to determine staleness.
+.PHONY: FORCE
+FORCE:
+
+# main entry point for running end to end tests. used by .github/workflows/e2e-tests.yml See test/e2e/README.md for details
+.PHONY: e2e-local
+e2e-local: bin/e2e-local.test test/e2e-local.image.tar
+	$(GINKGO) -nodes $(or $(NODES),1) -flakeAttempts 3 -randomizeAllSpecs $(if $(TEST),-focus '$(TEST)') -v -timeout 90m $< -- -namespace=operators -olmNamespace=operator-lifecycle-manager -dummyImage=bitnami/nginx:latest -kind.images=../test/e2e-local.image.tar
+
+# this target updates the zz_chart.go file with files found in deploy/chart
+# will only fire if a file in deploy/chart has been changed
+test/e2e/assets/chart/zz_chart.go: $(shell find deploy/chart -type f)
```

Comment:
Forcing this recipe to run always might save some future headaches. The prerequisite list can miss things like removed files or modification time shenanigans. I forget the details, but it definitely frustrated someone at some point -- the quick patch at the time was to avoid committing the generated file (https://github.com/operator-framework/operator-lifecycle-manager/pull/1454).

Reply:
Makes sense. I've updated the target to make if "phony" which forces its execution.